.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,56 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  workflow_dispatch:
+  pull_request:
+  create:
```

Comment:
@ChathurindaRanasinghe we don't need to create right? we identified an issue with that earlier. push would be enough